clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -76,15 +83,14 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 public class HeartbeatRequestManagerTest {
-    private long retryBackoffMs = DEFAULT_RETRY_BACKOFF_MS;
-    private int heartbeatIntervalMs = DEFAULT_HEARTBEAT_INTERVAL_MS;
-    private int maxPollIntervalMs = DEFAULT_MAX_POLL_INTERVAL_MS;
-    private long retryBackoffMaxMs = DEFAULT_RETRY_BACKOFF_MAX_MS;
+    private final long retryBackoffMs = DEFAULT_RETRY_BACKOFF_MS;
+    private final int heartbeatIntervalMs = DEFAULT_HEARTBEAT_INTERVAL_MS;
+    private final int maxPollIntervalMs = DEFAULT_MAX_POLL_INTERVAL_MS;
+    private final long retryBackoffMaxMs = DEFAULT_RETRY_BACKOFF_MAX_MS;
```

Comment:
nit: If there are final, could we remove them and directly use the constants where we need to?